library/src/main/java/com/rilixtech/CountryUtils.java

```diff
@@ -493,6 +493,10 @@ static int getFlagDrawableResId(Country country) {
         return R.drawable.flag_turks_and_caicos_islands;
       case "vi": //us virgin islands
         return R.drawable.flag_us_virgin_islands;
+      case "ss": //us virgin islands
```

Comment:
Thank you for the PR!. There are a slightly typo here: `//us virgin islands`. The comment didn't reflect the country code. You need to change the typos. I'll review and test the code tonight.

Reply:
I have corrected the typo error. Please Check it